### ISSUE TITLE
Create Nots for WCAG Failure Techniques

### ISSUE THREAD

**Issue by garcialo** [a11yNot/a11yNot.github.io#1]
Prioritize Failures for Success Criteria checked by the tools.

http://www.w3.org/WAI/GL/WCAG20-TECHS/failures.html

**Comment by garcialo** [comment]
Created most of these. Still need to make some custom files for some that specify, html/head/body tags, etc. and can't be embedded on a page.
